AdGuard Mail initial EULA (#101)
AdguardTeam/LegalDocs: adguard-mail.com/eula.html.md

1.3. If you have received, downloaded, and/or installed a beta version of AdGuard Mail, as identified by AdGuard Software Limited, AdGuard Software Limited hereby grants you a non-exclusive license to use the beta version of AdGuard Mail solely for testing purposes and only during the applicable testing period. This license is contingent upon your participation in the Beta testing program of AdGuard Software Limited and your compliance with all rules of the Beta testing program, as well as with all requirements, restrictions, and terms specified in this EULA. Any use of the beta version of AdGuard Mail for purposes other than testing or beyond the applicable testing period is strictly prohibited.

> 1.4. AdGuard Mail that AdGuard Software Limited may provide in accordance with clause 1.2 on a free basis may have a number of limitations, such as restrictions on the number of email aliases you can create and the number of recipients you can forward emails to. Information on these and other limitations is provided on the Website.

Reviewer: ```suggestion
1.4. AdGuard Mail, which AdGuard Software Limited may provide on a free basis in accordance with Section 1.2, may have a number of limitations, such as restrictions on the number of email aliases you can create and the number of recipients you can forward emails to. Information on these and other limitations is provided on the Website.
```

> 1.6. To use AdGuard Mail, you must install the application or visit the Website and sign in using your email address or sign up by creating a new account. You must provide a valid email address when signing up. You may also sign in with an email and password you have already registered on the Website or other Rightholder’s websites, such as adguard.com, previously.

Reviewer: ```suggestion
1.6. To use AdGuard Mail, you must install the Application or visit the Website and log in using your email address or sign up by creating a new account. You must provide a valid email address when signing up. You may also log in with an email and password that you have previously registered on the Website or other Rightholder’s websites, such as adguard.com.
```

1.7. We may modify or discontinue AdGuard Mail free services and benefits without notice at any time at our sole discretion.

> 1.8 Continued use of AdGuard Mail after any modifications to this EULA have been made will constitute your acceptance of those changes.

Reviewer: ```suggestion
1.8 Your continued use of AdGuard Mail after any modifications to this EULA have been made will constitute your acceptance of such modifications.
```

## 2. Restrictions

> 2.1. You may not make or distribute copies of the Software or electronically transfer the Software from one device to another or over a network.

Reviewer: ```suggestion
2.1. You may not make or distribute copies of the Software or transfer the Software from one device to another or over a network.
```

2.2. You may not alter, merge, modify, or adapt the Software, or decompile, reverse engineer, disassemble, or otherwise reduce the Software to a human-perceivable form, except to the extent permitted by applicable law notwithstanding this restriction or to the extent otherwise expressly permitted by the Rightholder.

2.3. You may not sell, resell, rent, lease, or sublicense the Software. Only authorized Partners of Rightholder are permitted to resell or sublicense the Software.

2.4. You may not modify the Software or create derivative works based upon the Software.

2.5. In the event that you fail to comply with this EULA, Rightholder may terminate all licenses to the Software, and you must destroy all copies of the Software.

2.6. Unless otherwise provided herein, you shall not when using the beta version of the Software

> (A) use the beta version of the Software for a purpose other than the sole purpose of testing the Software,

Reviewer: ```suggestion
(A) use the beta version of the Software for any purpose other than to test the Software,
```

(B) use the beta version of the Software if you do not participate in the Beta testing program of the Rightholder or if you violate the rules of the Beta testing program,

(C) distribute, share, send, or make the beta version of the Software otherwise available to third parties without prior written consent of the Rightholder, or

> (D) breach the confidentiality of the beta version of the Software, its functions, features, characteristics, design, user interface, documentation, or other related materials by publishing, sharing, sending, disclosing, or otherwise making available to third parties without prior written consent of the Rightholder.

Reviewer: ```suggestion
(D) breach the confidentiality of the beta version of the Software, its functions, features, characteristics, design, user interface, documentation, or other related materials by publishing, sharing, sending, disclosing, or otherwise making them available to third parties without prior written consent of the Rightholder.
```

> 2.7. You may not remove or modify any proprietary notice or labels from the Software or any output of the Software, including author attribution and copyright notices.

Reviewer: ```suggestion
2.7. You may not remove or modify any proprietary notices or labels from the Software or any output of the Software, including author attribution and copyright notices.
```

> 2.8. You may not use the AdGuard Mail application for any unauthorized or illegal purpose or activity under applicable international, national, and local laws in countries where the application is used and where the AdGuard Mail servers are located. This includes, but is not limited to:

Reviewer: ```suggestion
2.8. You may not use the AdGuard Mail application for any unauthorized or illegal purpose or activity under applicable international, national, and local laws in the countries where the application is used and where the AdGuard Mail servers are located. This includes, but is not limited to:
```

(A) Sending, uploading, or transmitting any materials or information of unacceptable content, including but not limited to offensive, defamatory, fraudulent, or harassing content, as well as content that violates any law or the rights of others.

(B) Infringing upon the intellectual property rights of others, including but not limited to patents, trademarks, trade secrets, copyrights, or other proprietary rights.

(C) Engaging in the transmission of unsolicited advertising, "spam" emails, or other deceptive practices, including misrepresenting yourself as another person or entity.

> 2.9. Only persons 18 years of age or older may use AdGuard Mail. You are not allowed to use AdGuard Mail if you are a minor, if you have been or are prohibited to access AdGuard Mail, or if your Account has been suspended or closed due to any reason.

Reviewer: ```suggestion
2.9. Only persons 18 years of age or older may use AdGuard Mail. You are not allowed to use AdGuard Mail if you are a minor, if you have been or are prohibited to access AdGuard Mail, or if your Account has been suspended or deleted due to any reason.
```

2.10. You may not use another person’s email address or misrepresent yourself as another person when signing up to create your Account or when logging in to use AdGuard Mail.

> 2.11. You may not download, install, use the Software or purchase the subscription to the Software if the country in which you reside has restricted or prohibited the use of Mail software, or the country belongs to E.U. or U.S. embargoed countries, or if you are a person to which shipment of the Software is prohibited by the export control laws, rules, regulations, restrictions and national security controls of the European Union, the United States, and other applicable foreign agencies.

Reviewer: ```suggestion
2.11. You may not download, install, use the Software or purchase the subscription to the Software if the country in which you reside restricts or prohibits the use of email software, or if that country is an EU or U.S. embargoed country, or if you are a person to whom the shipment of the Software is prohibited under the export control laws, rules, regulations, restrictions and national security controls of the European Union, the United States and other applicable foreign agencies.
```

> 2.12. In case of violation, we reserve the right to suspend your Account or limit access to AdGuard Mail at our sole discretion. Such suspension or access limitation may be implemented by AdGuard Mail instantly and without any notice. If your Account has been suspended, you can contact us for further information.

Reviewer: ```suggestion
2.12. In the event of a violation, we reserve the right to suspend your account or limit your access to AdGuard Mail at our sole discretion. Such suspension or restriction may be implemented by AdGuard Mail immediately and without notice. If your Account has been suspended, you may contact us for further information.
```

## 3. Personal data and Privacy Policy

3.1. AdGuard Mail collects and processes limited user data and has a strong Privacy Policy available at <https://adguard-mail.com/en/privacy.html> that explains in detail how AdGuard Software Limited handles your personal data in relation to your use of AdGuard Mail and your browsing of the Website.

3.2. Before using AdGuard Mail, you acknowledge that you have read and understood our Privacy Policy, which explains the types of information we collect and process, and what may happen to that information, including, without limitation, how such information is shared and used.

> By installing and using AdGuard Mail, you agree to the present Terms and the terms and conditions of data processing by AdGuard Software Limited as set out in our Privacy Policy available at the link mentioned above.

Reviewer: ```suggestion
By installing and using AdGuard Mail, you agree to be bound by these terms and conditions of data processing by AdGuard Software Limited as set out in our Privacy Policy, which is available at the link above.
```
